test/sequence/adapt_struct.cpp

```diff
@@ -123,6 +141,12 @@ namespace ns
         (BOOST_FUSION_ADAPT_AUTO, y)
     )
 
+    BOOST_FUSION_ADAPT_STRUCT(
+        ns::employee,
+        (std::string, name),
```

Comment:
Oh yes you are right. It's in else case when no variadics support, then this way of writing doesn't work, actually it should never compile when BOOST_PP_VARIADICS is disabled. I tested with BOOST_PP_VARIADICS enabled all the time for this PR.

I'll make myself a script to avoid forgetting such combinations. I'll make a PR to fix it.